ui/component/or-table/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@openremote/or-table",
+  "version": "1.0.0",
+  "description": "OpenRemote table",
+  "main": "dist/umd/index.bundle.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "npx tsc -b",
+    "test": "echo \"No tests\" && exit 0",
+    "prepublishOnly": "npx tsc -b && npx webpack"
+  },
+  "author": "OpenRemote",
+  "license": "AGPL-3.0-or-later",
+  "dependencies": {
+    "@material/data-table": "^5.1.0",
```

Comment:
`or-attribute-history` also uses the `@material/data-table`, should look to use `or-table` within that component in the future but for now please update the package version there also (hopefully no breaking changes)

Reply:
Done